test/sagemaker_tests/tensorflow/tensorflow2_training/integration/local/test_training.py

```diff
@@ -72,7 +74,11 @@ def test_mnist_cpu(sagemaker_local_session, docker_image, tmpdir, framework_vers
                     output_path=output_path,
                     training_data_path='file://{}'.format(
                         os.path.join(RESOURCE_PATH, 'mnist', 'data')))
-    _assert_files_exist_in_tar(output_path, ['my_model.h5'])
+    # _assert_checkpoint_exists_v2(output_path)
+    ############################################################################
+    #######TODO: This is something that needs to be confirmed from TF2.6 #######
```

Comment:
I have made this fix in which I am not using `_assert_files_exist_in_tar` function instead of  `_assert_checkpoint_exists_v2`. I want the change to be verified.

I am not sure why we were using [_assert_checkpoint_exists_v2](https://github.com/aws/deep-learning-containers/blob/2b199676aee4d5e52af4f53e2c7631095b20b13a/test/sagemaker_tests/tensorflow/tensorflow2_training/integration/local/test_training.py#L179) to search for checkpoint in the s3 bucket? I ask this because, the [`output_path`](https://github.com/aws/deep-learning-containers/blob/2b199676aee4d5e52af4f53e2c7631095b20b13a/test/sagemaker_tests/tensorflow/tensorflow2_training/integration/local/test_training.py#L67) variable looks like `file://{}`. As per the sagemaker estimator [documentation](https://sagemaker.readthedocs.io/en/stable/api/training/estimators.html), `output_path` with `file://{}` format saves the training results in the local folder. That being said, on investigation, I found out that 2 tar files, namely `model.tar.gz` and `output.tar.gz` are being saved successfully in the desired location. Once I untar `model.tar.gz`, I get a bunch of files and folders. One of the untarred files turns out to be a pickle file called `saved_model.pb`.

This might pass the test, but I am not sure if this is what the team desired. I would like the TF team to look into the changes I made in this commit and verify if the things are working as intended.

Reply:
It's because `_assert_checkpoints_exitsts_v2` is agnostic about the naming of the files in `output_path`, tensorflow might save the checkpoint with different conventions across different versions.

Reply:
Do we need to check the file in s3 bucket since we are saving the output locally?
We can just change [this](https://github.com/aws/deep-learning-containers/blob/2b199676aee4d5e52af4f53e2c7631095b20b13a/test/sagemaker_tests/tensorflow/tensorflow2_training/integration/local/test_training.py#L80) line to _assert_files_exist_in_tar(output_path, [])  such that it becomes agnostic in nature, right?